### ISSUE TITLE
404 when clicking on "See original" in a newsletter 

### ISSUE THREAD

**Issue by Neomac** [omnivore-app/omnivore#2355]
Hi, thank you so much for this project and the work you've put into it!

I tried to subscribe to a newsletter (https://maneetpaul.com/newsletter), but when I received the confirmation email, I couldn't confirm my email. The button was not visible in Omnivore: 

![Capture d’écran 2023-06-13 à 20 25 41](https://github.com/omnivore-app/omnivore/assets/3006818/5443b0cf-87a8-4449-84ad-50a5547f32fd)

So I tried the "See Original" button on the top of the page but I got a 404 on this URL: https://omnivore.app/no_url?q=9c7cbc38-6c8e-4e03-9005-8e98f8add9c4

 ![Capture d’écran 2023-06-13 à 20 27 03](https://github.com/omnivore-app/omnivore/assets/3006818/cd31036d-092e-4bac-ba81-fb139c3d30ca)

I already reported the formatting issue directly in the app.

**Comment by jacksonh** [comment]
Thanks, can you visit this page and see if the button is accessible? https://omnivore.app/settings/emails/recent

**Comment by jacksonh** [comment]
Ah we should be hiding the See Original button in this case, as it does not have a URL.

**Comment by Neomac** [comment]
> Thanks, can you visit this page and see if the button is accessible? https://omnivore.app/settings/emails/recent

Worked perfectly, thanks! Maybe, you could a link to the doc (https://docs.omnivore.app/using/inbox.html#debugging-issues-with-your-email-inbox) next to the report issue link at the bottom of the email. This way people can debug this type of issue them :). 

Thanks again for this great tool!

**Comment by jacksonh** [comment]
Ah smart idea!

**Comment by jacksonh** [comment]
Yes, we should add an option to see original contents of all messages.

**Comment by Podginator** [comment]
Closing due to age, and the self-hosted nature of Omnivore. Done as a batch action. If this issue still exists, please reopen it.
